src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2576,8 +2682,29 @@ public override bool IsRefLikeType
             }
         }
 
-        internal override string ExtensionName
-            => Name; // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Confirm implementation
+        internal override string ExtensionName // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Confirm implementation
```

Comment:
This was changed in `main` branch to a specific link:
```
        internal override string ExtensionName
            => Name; // Tracked by [https://github.com/dotnet/roslyn/issues/78963](https://github.com/dotnet/roslyn/issues/78963) : Revisit when adopting new metadata design with content-based type names
```
Consider using a PROTOTYPE comment or removing the link entirely since there's a PROTOTYPE in implementation.

Reply:
> Consider using a PROTOTYPE comment or removing the link entirely since there's a PROTOTYPE in implementation.

Removing